.github/CODEOWNERS

```diff
@@ -180,664 +200,664 @@
+/sdk/managementgroups/arm-managementgroups/ @qiaozha @dw511214992 @MaryGao
 /sdk/managementgroups/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/managementpartner/arm-managementpartner @qiaozha @dw511214992 @MaryGao
+/sdk/managementpartner/arm-managementpartner/ @qiaozha @dw511214992 @MaryGao
 /sdk/managementpartner/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/maps/arm-maps @qiaozha @dw511214992 @MaryGao
+/sdk/maps/arm-maps/ @qiaozha @dw511214992 @MaryGao
 /sdk/maps/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/mariadb/arm-mariadb @qiaozha @dw511214992 @MaryGao
+/sdk/mariadb/arm-mariadb/ @qiaozha @dw511214992 @MaryGao
 /sdk/mmariadbaps/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/marketplaceordering/arm-marketplaceordering @qiaozha @dw511214992 @MaryGao
+/sdk/marketplaceordering/arm-marketplaceordering/ @qiaozha @dw511214992 @MaryGao
 /sdk/marketplaceordering/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/mediaservices/arm-mediaservices @qiaozha @dw511214992 @MaryGao
+/sdk/mediaservices/arm-mediaservices/ @qiaozha @dw511214992 @MaryGao
 /sdk/mediaservices/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/migrate/arm-migrate @qiaozha @dw511214992 @MaryGao
+/sdk/migrate/arm-migrate/ @qiaozha @dw511214992 @MaryGao
 /sdk/migrate/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/mixedreality/arm-mixedreality @qiaozha @dw511214992 @MaryGao
+/sdk/mixedreality/arm-mixedreality/ @qiaozha @dw511214992 @MaryGao
 /sdk/mixedreality/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/monitor/arm-monitor @qiaozha @dw511214992 @MaryGao
+/sdk/monitor/arm-monitor/ @qiaozha @dw511214992 @MaryGao
 /sdk/monitor/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/monitor/arm-monitor-profile-2019-03-01-hybrid @qiaozha @dw511214992 @MaryGao
+/sdk/monitor/arm-monitor-profile-2019-03-01-hybrid/ @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/monitor/arm-monitor-profile-2020-09-01-hybrid @qiaozha @dw511214992 @MaryGao
+/sdk/monitor/arm-monitor-profile-2020-09-01-hybrid/ @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/msi/arm-msi @qiaozha @dw511214992 @MaryGao
+/sdk/msi/arm-msi/ @qiaozha @dw511214992 @MaryGao
 /sdk/msi/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/mysql/arm-mysql @qiaozha @dw511214992 @MaryGao
+/sdk/mysql/arm-mysql/  @qiaozha @dw511214992 @MaryGao
 /sdk/mysql/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/netapp/arm-netapp @qiaozha @dw511214992 @MaryGao
+/sdk/netapp/arm-netapp/ @qiaozha @dw511214992 @MaryGao
 /sdk/netapp/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Mgmt
-/sdk/network/arm-network @qiaozha @dw511214992 @MaryGao
-/sdk/network/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
+/sdk/network/arm-network/ @qiaozha @dw511214992 @MaryGao
```

Comment:
Do we need extra rule for ` sdk/network/arm-network-rest/` ?

Reply:
OK I need to fix these paths to end with `*/` instead of `/`, because they were used to match multiple directories before I introduced `/`.

Reply:
Those are like any other directory under sdk without owners they fallback to the leads until an owners is identified.

Reply:
Or if they were matching another path that we fixed by adding a "/" we could make those map to what they used to map to. 

These actually look like they are arm packages so we might want to consider a wildcard to map the all to the arm/mgmt owners.

Reply:
@qiaozha @dw511214992 @MaryGao can you let me know if you want to be owners of following paths?

```
/sdk/network/arm-network-rest/
/sdk/appservice/arm-appservice-rest/
/sdk/compute/arm-compute-rest/
/sdk/containerservice/arm-containerservice-rest/
/sdk/mysql/arm-mysql-flexible/
/sdk/servicefabric/arm-servicefabric-rest/
```

Currently their owner is @xirzec, via the `/sdk/` rule.

Reply:
sorry just see this comment, @konrad-jamrozik yes, you can put us as owner.

Reply:
@qiaozha @weshaggard @benbp
- https://github.com/Azure/azure-sdk-for-js/pull/24842